examples/viewer.js

```diff
@@ -36,16 +38,239 @@ let droppedFramePerArray = [];
 let videoBitRateArray = [];
 let audioRateArray = [];
 let timeArray = [];
+let chartHeight = 0;
+
+let metrics = {
```

Comment:
Nice work!

Let's add a comment explaining that these metrics are received over datachannel msgs from the C SDK sample (from pr description).

Reply:
I have explained the flow but not mentioned the C SDK in particular. I can add that